src/scss/generic/_heading.scss

```diff
@@ -151,6 +151,16 @@ h6,
     font-size: 12px;
   }
 
+  &--mini {
```

Comment:
@eduardojmatos Yes, I was in doubt of where the right place should be. Here I can use it as caption-title--mini. Inside the card scope I will have to test what the implementation would be. Do you prefer inside the card?